src/fastoad/openmdao/variables/variable.py

```diff
@@ -135,6 +136,19 @@ def __init__(self, name, **kwargs):
         if not self.description and self.name in self._variable_descriptions:
             self.description = self._variable_descriptions[self.name]
 
+    def get_val(self, new_units: Union[str, None] = None):
+        """Method used to get the variable value converted in the new_units"""
```

Comment:
Well, we already know it is a method 😉 

```suggestion
        """Returns the variable value converted in the `new_units`"""
```

Reply:
Modified in [b91d0ee](https://github.com/fast-aircraft-design/FAST-OAD/pull/570/commits/b91d0ee67ed3bb55ceabc5bbfa778fd9a5f7126f) 😅